main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"flag"
+	"os"
+
+	"github.com/medibloc/panacea-dep-consumer/server"
+)
+
+func main() {
+	httpPtr := flag.String("listen-addr", "", "http server listen address")
+	grpcPtr := flag.String("grpc-addr", "", "grpc server listen address")
+	chainIDPtr := flag.String("chain-id", "", "chain ID of Panacea blockchain")
+	flag.Parse()
+	if err := server.Run(*httpPtr, *grpcPtr, *chainIDPtr); err != nil {
```

Comment:
If you want to run using flags:
`./build/dep-consumer -listen-addr="127.0.0.1:8080" -grpc-addr="http://127.0.0.1:9090" -chain-id="..."`

Reply:
put this instruction to the readme. always think from the user's point of view.

Reply:
I'll add the `README` in other branch, so I'll mark TODO on it.